README.md

```diff
@@ -19,17 +19,27 @@ None of them is dealing with jira.
 - [ ] optional: insert build types (e.g. beta or production)
 
 ## Requirements
-- python 2.x
-- [jira-python](https://github.com/pycontribs/jira)
-    - install via `pip install jira`
```

Comment:
would actually keep that, it's easier than another extra file (with just one line)